### PR TITLE
Prepare 16.0.0

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

Ref stylelint/stylelint#6930

> Is there anything in the PR that needs further explanation?

Summary:
- Fix Admonition conversion for new GFM alerts. See:
  - https://github.com/stylelint/stylelint/pull/7293
  - https://docusaurus.io/docs/markdown-features/admonitions
  - https://github.com/orgs/community/discussions/16925
- Add migration guide to sidebars
- Fix broken links in `/user-guide/rules`

See each commit for details.

To-dos:
- [x] Update the `stylelint` and `stylelint-config-standard` versions in `devDependencies` (`package.json`)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -101,7 +101,7 @@
     "remark": "^15.0.1",
     "remark-cli": "^12.0.0",
     "remark-gfm": "^4.0.0",
-    "stylelint": "https://github.com/stylelint/stylelint/tarball/15.11.0",
+    "stylelint": "https://github.com/stylelint/stylelint/tarball/v16",
     "stylelint-config-standard": "^34.0.0",
```
Comment (Mouvedia) [inline package.json:105]: question

Is `stylelint-config-standard` already compatible?

Comment (ybiquitous) [reply package.json:105]: `stylelint-config-standard` needs to update `engines.node` and `peerDependencies.stylelint`, but it should not need to update `rules` in the config. So basically it's compatible.

**Comment by jeddy3** [comment]
Rule links go to GitHub:
https://deploy-preview-359--stylelint.netlify.app/user-guide/rules

Is there something amiss with the rewrite script?

**Comment by ybiquitous** [comment]
Thanks. I'll look into a cause.

**Review by jeddy3 — APPROVED** [review]

**Pull request merged**